.github/workflows/issues.yml

```diff
@@ -0,0 +1,55 @@
+name: Tackle Controls Issues Sync
```

Comment:
Could we change this to `Issues sync`? No need to mention the name of the project
```suggestion
name: Issues sync
```